jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/TopicDraft.java

```diff
@@ -32,27 +31,24 @@
 }, message = "{topicDraft.fields.not_null}")
 public class TopicDraft extends Entity {
 
-    @Size(min = Topic.MIN_NAME_SIZE, max = Topic.MAX_NAME_SIZE,
+    @Size(min = 1, max = Topic.MAX_NAME_SIZE,
```

Comment:
For instance, instead of using `deleteByUser` method of `TopicDraftDao`, we could add method `clean`, which will clean already existing entries. But I don't think that this help us simplify the design.